solvers/DeepCORAL.py

```diff
@@ -48,14 +48,23 @@ def get_estimator(self, n_classes, device, dataset_name, **kwargs):
         else:
             raise ValueError(f"Unsupported dataset: {dataset_name}")
         
+        lr_scheduler = LRScheduler(
+            policy='StepLR',
+            step_every='epoch',
+            step_size=1,
+            gamma=0.7
+        )
+        
+        
         net = DeepCoral(
             model,
-            optimizer=AdamW,
-            layer_name="feature_layer",
+            optimizer=Adadelta,
+            layer_name="fc1",
             batch_size=256,
             max_epochs=1,
```

Comment:
Two minor comments:
- by default in https://github.com/pytorch/examples/blob/main/mnist/main.py#L75 :  batch_size=64 (but bigger is better)
- you set `max_epochs=1` , I guess it is overwritten by the `param_grid` (i.e. `max_epochs=14`)